### PR TITLE
Fix devcontainer run command

### PR DESCRIPTION
**Pull request by tanya-borisova**
Pass GH_ORG_TOKEN correctly into postCreateCommand of the devcontainer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,8 +78,11 @@ be created in a private repository created from this template repository.
 
     The output will be used in the next step.
 
+1. Assing a `Application.ReadWrite.OwnedBy` role to the Service Principal you've just created.
```
Comment (tanya-borisova) [inline README.md:81]: Just a drive-by as I had to go through this

Comment (damoodamoo) [reply README.md:81]: this is covered in the `make ci-auth` in #95 now

Comment (t-young31) [inline README.md:81]:
```suggestion
1. Adding a `Application.ReadWrite.OwnedBy` role to the Service Principal you've just created. 
```

**Review by t-young31 — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (tanya-borisova) [reply README.md:81]: Cool, do you want me to remove this?
